File: backend/BulkApi/Models/DiscountScheme.cs
using System;
using System.Collections.Generic;

namespace BulkApi.Models
{
    public class DiscountScheme
    {
        public int DiscountSchemeId { get; set; }
        public int MinOrderQnty { get; set; }
        public double DiscountedPrice { get; set; }
        public DateTime? ExpiryDate { get; set; }
        public double DeliveryCharge { get; set; }

        //ProductFk
        public int ProductId { get; set; }
        public Product Product { get; set; }

        public List<Bid> Bids { get; set; }
    }
}

File: frontend/bulkApi/src/models/DiscountScheme.ts
import React from "react"
import { Product } from "./Product";
import { NONAME } from "dns";
import { Bid } from "./Bid";

export class DiscountScheme {
    discountSchemeId: number = 0;
    minOrderQnty: number = 0;
    discountedPrice: number = 0;
    expiryDate: Date | null = null;
    deliveryCharge: number = 0;

    productId: number = 0;
    product: Product | null = null;
    bids: Bid[] = [];

}